webapp/channels/src/components/advanced_text_editor/toggle_formatting_bar.test.tsx

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {fireEvent} from '@testing-library/react';
+import React from 'react';
+import {IntlProvider} from 'react-intl';
+
+import '@testing-library/jest-dom/extend-expect';
+import {renderWithContext} from 'tests/react_testing_utils';
+
+import ToggleFormattingBar from './toggle_formatting_bar';
+
+jest.mock('components/with_tooltip', () => {
+    return ({children}: { children: React.ReactNode }) => <div>{children}</div>;
+});
+
+describe('ToggleFormattingBar Component', () => {
+    it('should render correctly with default props', () => {
+        const {getAllByLabelText} = renderWithContext(
```

Comment:
we should be using `screen` from `tests/react_testing_utils` instead of getting it from the return value
```suggestion
        renderWithContext(..
        .
        .
        screen.getAllByLabelText(..
```